docs/src/organisms/accordion.js

```diff
@@ -831,7 +839,7 @@ export default class CollectionsAccordion extends React.Component {
                     <Accordion selected={controlledExampleSelectedIndex}>
                         <Accordion.Item>
                             <Accordion.Summary onClick={this._onClickAccordionItem}>
-                                <Header size="small">{'Option One'}</Header>
+                                <Header size="small">Option One</Header>
```

Comment:
This is what I was talking about [here](https://github.com/saddlebackdev/react-cm-ui/pull/147#discussion_r442274597).